content/en/docs/ops/best-practices/observability/index.md

```diff
@@ -13,14 +13,10 @@ The recommended approach for production-scale monitoring of Istio meshes with Pr
 is to use [hierarchical federation](https://prometheus.io/docs/prometheus/latest/federation/#hierarchical-federation)
 in combination with a collection of [recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/).
 
-In default deployments of Istio, a deployment of [Prometheus](http://prometheus.io) is
-provided for collecting metrics generated for all mesh traffic. This deployment of
-Prometheus is intentionally deployed with a very short retention window (6 hours). The
-default Prometheus deployment is also configured to collect metrics from each Envoy proxy
-running in the mesh, augmenting each metric with a set of labels about their origin (`instance`,
-`pod`, and `namespace`).
-
-While the default configuration is well-suited for small clusters and monitoring for short time horizons,
+Istio no longer ships with Prometheus in its default profile. To enable Prometheus please see
+the [Prometheus integration](../../../ops/integrations/prometheus/) addon documentation.
+
+While the `Quick Start` installation is well-suited for small clusters and monitoring for short time horizons,
```

Comment:
Not sure what `Quick Start` is. Maybe you are referring the the `Getting Started` installation. Or we could say `default configuration profile`.

Reply:
@douglas-reid PTAL

Reply:
I was referencing https://istio.io/latest/docs/ops/integrations/prometheus/#option-1-quick-start which is mentioned in the link to the Prometheus integration. The phrasing could be better.

Reply:
Should we call it `sample` install ?

Reply:
We no longer install Prometheus in the default nor in the demo profile. For the users to test prometheus they have to follow the addons integration page. It's not difficult, but no longer an option for testing out the functionality easily.

Reply:
I'm wondering if this paragraph would be a better tip/info on the prometheus add-on page. We do have a very small blurb there.

Reply:
Moved to the addons page for Prometheus. Combined Frank's suggestion too.